### PR TITLE
Add multiarch manifests for Java images

### PR DESCRIPTION
**Pull request by Sineaggi**
Adds a few more images with multiarch manifests. Based off of the previously tagged images

### CODE REVIEW

**Comment by loosebazooka** [comment]
Hey sorry this has just been low priority for us, I'll take a look sometime soon.

**Comment by timja** [comment]
Any chance of this @loosebazooka, with Apple only releasing ARM laptops now a lot of new people don't have intel processors.

Would be great to get this (we can look at rebuilding internally if required)

File: cloudbuild_docker.sh [hunk 1]
```diff
@@ -57,4 +57,7 @@ docker manifest create gcr.io/$PROJECT_ID/nodejs:debug \
    gcr.io/$PROJECT_ID/nodejs:debug-arm64
 docker manifest push gcr.io/$PROJECT_ID/nodejs:debug
 
-# No Java? Only available on amd64, we do not need to generate a manifest list
+docker_manifest gcr.io/$PROJECT_ID/java:11-debian11:nonroot "amd64 arm64"
```
Comment (loosebazooka) [inline cloudbuild_docker.sh:60]: these tags seem off.

File: BUILD [hunk 2]
```diff
@@ -203,6 +203,17 @@ JAVA = {
     "{REGISTRY}/{PROJECT_ID}/java-debian11/jetty:java11-debug": "//java/jetty:jetty_java11_debug_debian11",
 }
 
+JAVA.update({
+    "{REGISTRY}/{PROJECT_ID}/java-debian11:" + tag_base + "-" + arch: "//base:" + label + "_" + user + "_" + arch + "_debian11"
+    for arch in ARCHITECTURES
```
Comment (loosebazooka) [inline BUILD:208]: should be just `arm64` and `amd64`

File: BUILD [hunk 3]
```diff
@@ -203,6 +203,17 @@ JAVA = {
     "{REGISTRY}/{PROJECT_ID}/java-debian11/jetty:java11-debug": "//java/jetty:jetty_java11_debug_debian11",
 }
 
+JAVA.update({
+    "{REGISTRY}/{PROJECT_ID}/java-debian11:" + tag_base + "-" + arch: "//base:" + label + "_" + user + "_" + arch + "_debian11"
```
Comment (loosebazooka) [inline BUILD:207]: uh, this doesn't really make sense? you're not supposed to be building base.

**Review by loosebazooka — COMMENTED** [review]
Lots of issues here.

**Comment by loosebazooka** [comment]
Oops, did not mean to be rude (or sound it). I'll post meaningful corrections tomorrow

**Comment by Sineaggi** [comment]
> Oops, did not mean to be rude (or sound it). I'll post meaningful corrections tomorrow

No worries, you're supposed to enjoy your day off ;)

**Comment by loosebazooka** [comment]
I ended up reworking the java image tagging as it was getting out of hand. See #902

**Pull request closed without merging**